### ISSUE TITLE
run-workers 2 ignores --profile in child processes

### ISSUE THREAD

**Issue by pablopaul** [Codeception/CodeceptJS#1968]
#### What are you trying to achieve?

Use `--profile` with `run-workers`.

#### What do you get instead?

Provided profile seems to be ignored.

See https://gitlab.com/codeceptjs/codeceptjs-bug-report-template/-/jobs/322868867 where selenium can be found if I use `run` instead of `run-workers`.

### Details

* CodeceptJS version: 2.3.3
* NodeJS Version: 12

**Comment by orihomie** [comment]
I confirm a bug

**Comment by orihomie** [comment]
As far as I see - its enough to add something like this to run-workers command file:

```js
module.exports = function (workers, options) {
  satisfyNodeVersion(
    '>=11.7.0',
    'Required minimum Node version of 11.7.0 to work with "run-workers"',
  );
  const { Worker } = require('worker_threads');
  ...
  process.profile = options.profile; // possible fix
  ...
```

**Comment by pablopaul** [comment]
@orihomie how about adding a PR with this change?

**Comment by koushikmohan1996** [comment]
I think this is fixed in #1971 . @DavertMik Can you confirm?

**Comment by pablopaul** [comment]
My current workaround here is to avoid usage of `--profile` and instead use a separate config file with `-c configs/…`

**Comment by pablopaul** [comment]
Should be fixed with newest release 2.6 - use process.env.profile instead of process.profile.
